src/app.js

```diff
@@ -1,69 +1,68 @@
-const base64 = require('base-64')
-
-    // $('#sign-up-form').submit(function (event) {
-    //   event.preventDefault();
-    //   $('#output-box').text('HELLO WORLD');
-    // })
-
-//   event.preventDefault();
-//   const ajaxSettings = {
-//     method: 'POST'
-//   };
-//   let username = $('#sign-up > .username').val()
-//   let password = $('#sign-up > .password').val()
-//   let bio = $('#sign-up > .content').val()
-//   let user = {
-//     username: username,
-//     password: password,
-//     bio: bio
-//   }
-//   $.ajax(`http://localhost:3000/signup`, ajaxSettings)
-//     .send(user)
-//     .then(data => {
-//       $('#output-box').text(data)
-//       // data
-//     });
-// })
-
+// const base64 = require('base-64')
```

Comment:
I'm thinking we can just delete this file since we met MVP and all the code here is commented out anyway